CONTRIBUTING.md

```diff
@@ -22,8 +22,7 @@ To revert back to your original version of Tribler, [download the installer agai
     * The version of Tribler that you are using, if you are running from a branch, branch name and commit ID.
     * The OS and version you are running.
     * Step by step instructions to reproduce the issue in case it's a bug.
-    * Attach Tribler's log file found in `%APPDATA%` on Windows or on
-      `~/.Tribler/` on Linux/OSX (have a look at its contents before posting it
+    * Attach Tribler's log file found in `%APPDATA%` on Windows, `~/.Tribler/` on Linux or the crash log found in `~/Library/Logs/DiagnosticReports` on OS X (have a look at its contents before posting it
```

Comment:
I've started Tribler some minutes ago and I see zero entries regarding stdout/stderr of Tribler in `/var/log/system.log`.